dora/core/server/master/src/test/java/alluxio/master/meta/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -99,18 +98,10 @@ public final class AlluxioMasterRestServiceHandlerTest {
   private static final long UFS_SPACE_USED = 25L;
   private static final long UFS_SPACE_FREE = 75L;
   private static final String TEST_PATH = "test://test/";
-  private static final Map<String, Long> WORKER1_TOTAL_BYTES_ON_TIERS =
-      ImmutableMap.of(Constants.MEDIUM_MEM, 10L,
-      Constants.MEDIUM_SSD, 20L);
-  private static final Map<String, Long> WORKER2_TOTAL_BYTES_ON_TIERS =
-      ImmutableMap.of(Constants.MEDIUM_MEM,
-      1000L, Constants.MEDIUM_SSD, 2000L);
-  private static final Map<String, Long> WORKER1_USED_BYTES_ON_TIERS =
-      ImmutableMap.of(Constants.MEDIUM_MEM, 1L,
-      Constants.MEDIUM_SSD, 2L);
-  private static final Map<String, Long> WORKER2_USED_BYTES_ON_TIERS =
-      ImmutableMap.of(Constants.MEDIUM_MEM, 100L,
-      Constants.MEDIUM_SSD, 200L);
+  private static final Map<String, Long> WORKER1_TOTAL_BYTES_ON_TIERS = new HashMap<>();
```

Comment:
nit: `ImmutableMap.of()`. I'm probably removing them soon anyways.

Reply:
you mean still use the `ImmutableMap.of()` ?